linked_list.py

```diff
@@ -0,0 +1,60 @@
+
+#import numpy as np
+
+# insertAt
+# deleteAt
+# first
+# last
+
+class Node:
+    def __init__(self, data=None):
+        self.next = None
+        self.data = data
+
+class LinkedList:
```

Comment:
Implement the following APIs as well.  

https://docs.python.org/3/tutorial/datastructures.html
list.insert(i, x)
list.remove(x)
list.pop([i])
list.index(x[, start[, end]])
list.count(x)

Reply:
Done